GETTING_STARTED.md

```diff
@@ -1,109 +1,86 @@
 ```python
-# Authentication with API key
-client = C8Client(protocol='https', host='gdn.macrometa.io', port=443,
- apikey=<your api key>)
+client.get_document(collection='employees', document={'_key': 'Han'})
 ```
 
-```python
-from c8 import C8Client
-import time
-import warnings
-
-warnings.filterwarnings("ignore")
-federation = "gdn.paas.macrometa.io"
-demo_tenant = "mytenant@example.com"
-demo_fabric = "_system"
-demo_user = "user@example.com"
-demo_user_name = "root"
-demo_stream = "demostream"
-collname = "employees"
-# --------------------------------------------------------------
-print("Create C8Client Connection...")
-client = C8Client(protocol='https', host=federation, port=443,
-                  email=demo_tenant, password='XXXXX',
-                  geofabric=demo_fabric)
-
-# --------------------------------------------------------------
-print("Create Collection and insert documents")
-
-if client.has_collection(collname):
-    print("Collection exists")
-else:
-    client.create_collection(name=collname)
-
-# List all Collections
-coll_list = client.get_collections()
-print(coll_list)
-
-# Filter collection based on collection models DOC/KV/DYNAMO
-colls = client.get_collections(collection_model='DOC')
-print(colls)
-
-# Get Collecion Handle and Insert
-coll = client.get_collection(collname)
-coll.insert(
-    {'firstname': 'John', 'lastname': 'Berley', 'email': 'john.berley@macrometa.io'})
-
-# insert document
-client.insert_document(collection_name=collname,
-                       document={'firstname': 'Jean', 'lastname': 'Picard',
-                                 'email': 'jean.picard@macrometa.io'})
-doc = client.get_document(collname, "John")
-print(doc)
-
-# insert multiple documents
-docs = [
-    {'firstname': 'James', 'lastname': 'Kirk', 'email': 'james.kirk@macrometa.io'},
-    {'firstname': 'Han', 'lastname': 'Solo', 'email': 'han.solo@macrometa.io'},
-    {'firstname': 'Bruce', 'lastname': 'Wayne', 'email': 'bruce.wayne@macrometa.io'}
-]
+## Update Document
+The [update](https://macrometa.com/docs/queryworkers/c8ql/got-tutorial/c8ql-crud#update-documents) operation can partially update a document in a collection. The document must contain the attributes and values to be updated and the _key attribute to identify the document to be updated. For this example the email for *Han* will be updated.
 
-client.insert_document(collection_name=collname, document=docs)
+```python
+client.update_document(collection_name='employees', document={'_key': 'Han', 'email': 'han@updated_macrometa.io'})
+```
 
-# insert documents from file
-client.insert_document_from_file(collection_name=collname,
-                                 csv_filepath="/home/user/test.csv")
+## Delete Document
+The [delete](https://macrometa.com/docs/queryworkers/c8ql/got-tutorial/c8ql-crud#delete-documents) operation is used to remove a document from a collection. For this example *Han* will be removed from the **employees** collection.
```

Comment:
same c8ql point as above

Reply:
I think it's important to refer to any link in our doc, but sometimes it's a bit harder to find a narrowed section of a tutorial. for example here is https://macrometa.com/docs/collections/documents/document-store-data#delete-document , It doesn't say something related to the example. That's why I mentioned the C8QL

Reply:
Yes I know, so you don't need to actually refer to a specific doc here.. you can simply explain the basic operations using your example like we need to pass so and so as input and then so and so doc gets deleted.. We don't need to provide reference everywhere.. c8ql reference will not be the right approach here.. doc references are needed to explain the topic in general say what are documents what is a document collection what all we can do with document collection .. just to explain the theory of document collection.. But the basic operations we can explain ourselves in a line or two using an example..